docs/static/releasenotes.asciidoc

```diff
@@ -11,6 +12,49 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-1-1]]
+=== Logstash 6.1.1 Release Notes
+*  There are no user-facing changes in Logstash core in this release.
+
+[float]
+==== Input Plugins
+
+*`Beats`*:
+
+* Fix issue with close_wait connections, making sure keep alive sent back to the client. (https://github.com/logstash-plugins/logstash-input-beats/pull/272[#272])
+
+*`HTTP`*:
+
+* In the event that all webserver threads are busy this plugin will now return a 429, busy, error. (https://github.com/logstash-plugins/logstash-input-http/pull/75[#75])
+
```

Comment:
"In the event that" can just be "If". I also recommend avoiding the future tense unless it's absolutely necessary. I'd change this to:

"If all webserver threads are busy, the plugin now returns status code 429."

The phrasing here is a little problematic compared to standard HTTP status code terminology. The implication is that it's a server error (which would be a 5xx error like 503 Service Unavailable), but the 429 status code is a client error. It kind of makes sense, because you're trying to get them to back off on the request rate. But, saying a 429 code = "Busy" isn't consistent with normal usage--I'd just avoid the issue by dropping the short description at the end of the sentence.